roles/aodh/templates/etc/aodh/aodh.conf

```diff
@@ -35,17 +37,18 @@ connection = mysql+pymysql://aodh:{{ secrets.db_password }}@{{ endpoints.db }}/a
 signing_dir = /var/cache/aodh/api
 auth_url = {{ endpoints.keystone.url.admin }}
 auth_uri = {{ endpoints.keystone.url.internal }}
-project_domain_id = default
-user_domain_id = default
-project_name = service
-username = aodh
-password = {{ secrets.service_password }}
+admin_tenant_name = service
+admin_user = aodh
+admin_password = {{ secrets.service_password }}
 memcached_servers = {{ hostvars|ursula_memcache_hosts(groups, memcached.port) }}
+cafile = {{ aodh.cafile }}
+identity_uri = {{ endpoints.keystone.url.admin }}
```

Comment:
These read like old versions of arguments, is aodh using an older keystone library to handle these arguments?